<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vRenderer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="m_leftWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(194, 194, 194);
border-right-color: rgb(254, 57, 66); 
font: 13pt &quot;Helvetica Neue&quot;;</string>
    </property>
    <layout class="QVBoxLayout" name="m_leftBar">
     <item>
      <layout class="QVBoxLayout" name="m_sceneViewLayout">
       <item>
        <widget class="QLabel" name="m_sceneLabel">
         <property name="styleSheet">
          <string notr="true">font: 13pt &quot;Helvetica Neue&quot;;</string>
         </property>
         <property name="text">
          <string>Scene</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="m_sceneTreeView">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_loadMeshBtn">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 700 13pt &quot;Helvetica Neue&quot;;
border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedKingdom"/>
         </property>
         <property name="text">
          <string>Load mesh</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="m_envLayout">
       <item>
        <widget class="QLabel" name="m_envLabel">
         <property name="text">
          <string>Environment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_envHDRI">
         <property name="text">
          <string>HDRI</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_envCornell">
         <property name="text">
          <string>Cornell box</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_loadHDRBtn">
         <property name="styleSheet">
          <string notr="true">font: 700 13pt &quot;Helvetica Neue&quot;;
border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Load HDRI</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_hdrLabel">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>No environment map loaded</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="m_testSpace">
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Material</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_loadDiffuseTextureBtn">
         <property name="text">
          <string>Diffuse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_diffuseLabel">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>No diffuse texture loaded</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_loadNormalTextureBtn">
         <property name="text">
          <string>Normal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_normalLabel">
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>No normal texture loaded</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_loadSpecularTextureBtn">
         <property name="text">
          <string>Specular</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_specularLabel">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>No specular texture loaded</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="m_modifiersLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="m_fresnelLabel">
           <property name="text">
            <string>Fresnel coef</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="m_fresnelNumber">
           <property name="value" stdset="0">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="m_fresnelSlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="m_fresnelPowLabel">
           <property name="text">
            <string>Fresnel pow</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="m_fresnelPowNumber">
           <property name="smallDecimalPoint">
            <bool>false</bool>
           </property>
           <property name="value" stdset="0">
            <double>3.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="m_fresnelPowSlider">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="m_rightWidget" native="true">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>0</y>
      <width>200</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(194, 194, 194);
border-right-color: rgb(254, 57, 66); 
font: 13pt &quot;Helvetica Neue&quot;;</string>
    </property>
    <layout class="QVBoxLayout" name="m_rightBar">
     <item>
      <layout class="QVBoxLayout" name="m_cameraLayout">
       <item>
        <widget class="QLabel" name="m_cameraLabel">
         <property name="styleSheet">
          <string notr="true">font: 13pt &quot;Helvetica Neue&quot;;</string>
         </property>
         <property name="text">
          <string>Camera</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="m_fovLabel">
           <property name="text">
            <string>FOV</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="m_fovNumber">
           <property name="value" stdset="0">
            <double>75.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="m_fovSlider">
         <property name="minimum">
          <number>45</number>
         </property>
         <property name="maximum">
          <number>130</number>
         </property>
         <property name="sliderPosition">
          <number>75</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="m_fxaaLabel">
         <property name="text">
          <string>FXAA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_useFXAA">
         <property name="text">
          <string>Enabled</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="m_fxaaSoftness">
           <property name="text">
            <string>Softness</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="m_fxaaSoftnessNum">
           <property name="value" stdset="0">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="m_fxaaSoftnessSlider">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="m_fxaaSubpixQuality">
           <property name="text">
            <string>Subpix qlty</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="m_fxaaSubpixQualityNum">
           <property name="value" stdset="0">
            <double>0.750000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="m_fxaaSubpixQualitySlider">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>75</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="m_fxaaSubpixEdgeThreshold">
           <property name="text">
            <string>Edge thld</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="m_fxaaSubpixEdgeThresholdNum">
           <property name="value" stdset="0">
            <double>0.166000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="m_fxaaSubpixEdgeThresholdSlider">
         <property name="minimum">
          <number>63</number>
         </property>
         <property name="maximum">
          <number>333</number>
         </property>
         <property name="value">
          <number>166</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="m_brdfLayout">
       <item>
        <widget class="QLabel" name="m_sphereLabel">
         <property name="styleSheet">
          <string notr="true">font: 13pt &quot;Helvetica Neue&quot;;</string>
         </property>
         <property name="text">
          <string>Example Sphere</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_useSphere">
         <property name="text">
          <string>Enable sphere</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Merl BRDF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_useBRDF">
         <property name="text">
          <string>Use MERL BRDF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_loadBRDFBtn">
         <property name="text">
          <string>Load BRDF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_brdfLocation">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>No BRDF loaded</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="m_renderWidget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>720</width>
      <height>720</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="m_renderWidgetLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
